dotcom-rendering/scripts/check-node-versions.mjs

```diff
@@ -39,7 +39,7 @@ const requiredNodeVersionMatches =
 		},
 		{
 			filepath: '../apps-rendering/riff-raff.yaml',
-			pattern: /^ +Recipe: .+-mobile-node(\d+\.\d+\.\d+).*$/m,
+			pattern: /^ +Recipe: apps-rendering.*-node-(\d+\.\d+\.\d+).*?$/m,
```

Comment:
did you change the pattern in AMIgo?

Reply:
Yep: https://amigo.gutools.co.uk/recipes/apps-rendering-ARM-jammy-node-20.12.2/bakes/1